src/scalar_impl.h

```diff
@@ -55,6 +55,12 @@ static void secp256k1_scalar_order_get_num(secp256k1_num *r) {
 }
 #endif
 
+static int secp256k1_scalar_set_b32_seckey(secp256k1_scalar *r, const unsigned char *bin) {
+  int overflow;
+  secp256k1_scalar_set_b32(r, bin, &overflow);
+  return (!overflow) & (!secp256k1_scalar_is_zero(r));
```

Comment:
nit: indentation
I only point it out here but it's wrong in many functions you added.

Reply:
Ugh, fixed, but only saw this in one other function (run_scalar_set_b32_seckey_tests)